pallets/thea-executor/src/lib.rs

```diff
@@ -520,9 +562,8 @@ pub mod pallet {
 			let deposits: Vec<Deposit<T::AccountId>> =
 				Decode::decode(&mut &payload[..]).map_err(|_| Error::<T>::FailedToDecode)?;
 			for deposit in deposits {
-				<ApprovedDeposits<T>>::mutate(&deposit.recipient, |pending_deposits| {
-					pending_deposits.push(deposit.clone())
-				});
+				// Execute Deposit
+				Self::execute_deposit(deposit.clone())?;
 				Self::deposit_event(Event::<T>::DepositApproved(
```

Comment:
ApprovedEvent is emitted after the claim event (within Self::execute_deposit). This is breaking indexers as the approved event happened before the claim in the previous code. If claim event is obsolete now, better to remove it.

Reply:
Claim event is there pending deposits before upgrade, if that's not there, we can delete it now. In Fisherman model, claim process is not there anymore.

Reply:
As discussed with @nuel77, we are going to keep it for sometime.